src/Resources/services.yml

```diff
@@ -139,9 +139,6 @@ services:
   jaeger.span.handler.background:
     class: Jaeger\Symfony\Bridge\BackgroundSpanHandler
     arguments: ['@jaeger.tracer']
-  jaeger.span.handler.gloabal:
-    alias: jaeger.span.handler.global
-    deprecated: 'The "%alias_id%" service is deprecated. Use "jaeger.span.handler.global".'
```

Comment:
Symfony changed recommended style for deprecation aliases and there is no way to use deprecations in yaml compatible with 4 and 5 versions (without deprecations)
https://symfony.com/doc/current/service_container/alias_private.html#deprecating-service-aliases